testsuite/generate-format/src/libra.rs

```diff
@@ -55,6 +55,10 @@ pub fn get_registry() -> Result<Registry> {
     // 2. Trace the main entry point(s) + every enum separately.
     tracer.trace_type::<contract_event::ContractEvent>(&samples)?;
     tracer.trace_type::<language_storage::TypeTag>(&samples)?;
+    tracer.trace_type::<transaction::MetadataType>(&samples)?;
```

Comment:
OK so technically you need to list `GeneralMetadata`, `MetadataType ` and `TravelRuleMetadata` and the rest will follow (in a more future-proof way than the current list).